src/Microsoft.IdentityModel.Protocols.WsFederation/WsFederationMetadataSerializer.cs

```diff
@@ -28,6 +28,7 @@
 using System;
 using System.Security.Cryptography.X509Certificates;
 using System.Xml;
+using Microsoft.IdentityModel.Logging;
 using Microsoft.IdentityModel.Tokens;
 using Microsoft.IdentityModel.Xml;
 using static Microsoft.IdentityModel.Logging.IdentityModelEventSource;
```

Comment:
Check if you still need any of these 'using static'

Reply:
Got rid of all unnecessary 'using' statements in the files that were modified.